### DISCUSSION TITLE
Customizing login pages

### DISCUSSION THREAD

**Discussion by TegarAditya** [andrewdwallo/filament-companies#49] [Q&A] [answered]
Is it possible to modify the login pages?

**Comment by andrewdwallo** [comment, marked as answer]
Yes you can modify. Everything is connected through Fortify. What are you specifically wanting to modify?

**Reply by TegarAditya** [reply]
I'm trying to disable the registration feature via Fortify.  but when deactivated, the login page cannot be opened because at that time the route to register was not defined.  Maybe it's because of the link to the register route that's on the login page?

**Reply by andrewdwallo** [reply]
No you should be able to just go to the login page, but I will check and make sure when I get home.

**Reply by andrewdwallo** [reply]
> I'm trying to disable the registration feature via Fortify. but when deactivated, the login page cannot be opened because at that time the route to register was not defined. Maybe it's because of the link to the register route that's on the login page?

Why are you trying to disable registration? Are you replacing registration with something else? How are you supposed to login if your credentials are not registered? I would take note that almost all of the features of this package are irrelevant to use if you do not have registration... 

If you want to only modify the view of the register page, then just change the design by copying the view from my package's vendor folder into your application. 

Here are some resources that may help you as well:
https://jetstream.laravel.com/2.x/features/authentication.html
https://jetstream.laravel.com/2.x/features/registration.html

**Reply by andrewdwallo** [reply]
> I'm trying to disable the registration feature via Fortify. but when deactivated, the login page cannot be opened because at that time the route to register was not defined. Maybe it's because of the link to the register route that's on the login page?

I now see what the issue is... I will fix it but just note if you want to replace the register page with a different one it will probably work if you just create another route inside of the `web.php` file pointing to `/register` route. If you want a different route for registration other than `/register` you will probably need to modify the view pages like I mentioned earlier.

**Reply by andrewdwallo** [reply]
Issue resolved in latest release: https://github.com/andrewdwallo/filament-companies/releases/tag/v1.2.5
Please update and let me know if there is anything else you are having issues with. Thanks!

**Reply by TegarAditya** [reply]
> Why are you trying to disable registration? Are you replacing registration with something else? How are you supposed to login if your credentials are not registered? I would take note that almost all of the features of this package are irrelevant to use if you do not have registration...
> 
> If you want to only modify the view of the register page, then just change the design by copying the view from my package's vendor folder into your application.
> 
> Here are some resources that may help you as well: https://jetstream.laravel.com/2.x/features/authentication.html https://jetstream.laravel.com/2.x/features/registration.html

Initially I wanted to deactivate the registration feature because the application I was going to make was only for internal institutions so I wanted to turn off the independent registration of users and replace it with custom member management. Sorry for not mentioning this earlier.

**Reply by TegarAditya** [reply]
> I now see what the issue is... I will fix it but just note if you want to replace the register page with a different one it will probably work if you just create another route inside of the `web.php` file pointing to `/register` route. If you want a different route for registration other than `/register` you will probably need to modify the view pages like I mentioned earlier.

Thank you for your response. I just want to turn it off then use something like CRUD member on the admin page. The update you provided was very helpful in resolving this issue. Now I can turn `/register` route on and off easily. Once again Thank you very much, great job.

**Reply by andrewdwallo** [reply]
No problem, thanks.
